### ISSUE TITLE
Change variable names to symbols

### ISSUE THREAD

**Issue by dpsanders** [JuliaDiff/TaylorSeries.jl#147]
- [ ] Use `:x` instead of `"x"`
- [ ] Provide a `Dict` to look up which number variable a given symbol corresponds to.

**Comment by lbenet** [comment]
I don't mind the first option. 

Regarding the second one, I think we can get it directly as `look_up(:x)` with the function
```julia
lookup(s::Symbol) = find(v -> v==s, TaylorSeries._params_TaylorN_.variable_symbols)
```
It would return a vector with the index where the symbol is found, or a zero-length vector, if the symbol doesn't exist.

**Comment by lbenet** [comment]
This is only partially addressed by #154, shall we close this, or keep it open for the future?

**Comment by lbenet** [comment]
Closing, since this is somehow provided by #154 and #155
